### ISSUE TITLE
Sega Genesis/Master Drive /Cart - D40 Pin Issue

### ISSUE THREAD

**Issue by dannygonzalez0861** [sanni/cartreader#653]
Hi Sanni,

I think I might have just discovered an issue with the six-slot PCB Rev. 3.

I have been trying, w/o success, for the past week to get Mega Drive/Genesis Carts to read but the moment I plug one in and try to turn the reader on, it just stays off. I disconnect the game and try again and the reader powers on like usual. If the reader is powered on and I try to connect the game, it just shorts out and turns off.

I had a bit of free time and decided to see if I could find a short on my PCB but did not find a defect from my soldering. Well after a few hours of testing/probing for shorts and looking over your schematics/ Sega PCBs I physically have, I noticed that D40 which has continuity to VCC goes connected to the /cart pin of the genesis port which on all of my games is actually a ground pin.

I confirmed the result by applying a piece of tape to the /cart pin to avoid getting VCC to the pin as it shares ground on the game itself and the reader stays on as well as successfully pulls the ROM and passes the checksum.

Can you confirm if this is a one-off or a design issue?

**Comment by dannygonzalez0861** [comment]
![PXL_20221209_150010929](https://user-images.githubusercontent.com/61332341/206784974-72b4b7fd-58be-454a-9892-c76f855cc7ae.jpg)
![PXL_20221209_150021108](https://user-images.githubusercontent.com/61332341/206784975-4d8f3421-1084-481a-a13f-c4eb432c9cd3.jpg)
![PXL_20221209_150100867](https://user-images.githubusercontent.com/61332341/206784976-acb38d14-5b0f-4204-846b-964513859783.jpg)

**Comment by Ancyker** [comment]
D40 should not have continuity to VCC.

**Comment by Ancyker** [comment]
I was looking over the PCB layouts and the only place I see D40 ever getting close enough to VCC that it could accidentally be bridged is on the CIC adapter:

![image](https://user-images.githubusercontent.com/7105333/206812649-4016f5ae-f825-4d14-94d6-5eff06ca94bf.png)

**Comment by dannygonzalez0861** [comment]
I can't seem to find where the short on this board is. It is not on the CIC unfortunately, I disconnected the board as a quick test and the short still exists. I half-built a second unit over the weekend with the spare parts I had and it does not present these issues. It must be a one-off problem with this PCB.

Thanks for the help!

**Comment by Ancyker** [comment]
I think I had a similar issue with my first one. It was caused by a bridge
under one of the headers. You could try reflowing all the headers to see if
it helps.

**Comment by dannygonzalez0861** [comment]
Well, i though i would share what I found was the issue after all. I have no idea how this happened but the header for the SNES_CIC PCB was shorted almost completely. Not sure if I had a few too many that day but here is what I found when I started to disect the PCB, hope you all can laugh at my pain LMAO
![PXL_20221228_231752540](https://user-images.githubusercontent.com/61332341/210080896-0e3b93d7-ef78-4e8e-99a1-bbd296d7415d.jpg)
![PXL_20221228_231805828](https://user-images.githubusercontent.com/61332341/210080897-b57427cd-a7d5-4268-a715-d6e035165955.jpg)

@Ancyker your help was much appreciated, all of your ideas were indeed acurate and I just needed to fins exactly where the isse was.

Thanks!
